src/GovUk.Education.ExploreEducationStatistics.Common/Model/Chart/ChartDataSet.cs

```diff
@@ -14,9 +14,18 @@ public class ChartDataSet
         public int? Order;
 
         // TODO EES-1649 Migrate data set configs to legend item configs
+        // After EES-1649 migration, could consider removing ChartItemDataSet?
         public ChartDataSetConfiguration Config;
     }
 
+    public class ChartItemDataSet
```

Comment:
`ChartItemDataSet` is a bit of weird name. Would suggest `ChartBaseDataSet` and `ChartDataSet` could just also extend from it i.e.

```cs
public class ChartBaseDataSet
{
    public Guid Indicator;
    public List<Guid> Filters = new();
    public ChartDataSetLocation Location;
    public string TimePeriod;
}

public class ChartDataSet : ChartBaseDataSet
{
    public int? Order;

    // TODO EES-1649 Migrate data set configs to legend item configs
    // After EES-1649 migration, could consider removing ChartItemDataSet?
    public ChartDataSetConfiguration Config;
}
```